source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -0,0 +1,159 @@
+    { "_id": { "$oid": "606dfe52d506b97c576b22c2" }, "color": "green", "qty": 0 }
+    { "_id": { "$oid": "606dfe52d506b97c576b22c3" }, "color": "black", "qty": 8 }
+
+.. _update-operation:
+
+Update
+------
+
+To update one or more documents, create a ``Bson`` that specifies
+``Updates`` with a field and value that describe the change. 
+
+The ``Bson`` for ``Updates`` uses the following format:
+
+.. code-block:: java
+
+    Bson update = Updates.<update operator>("<field>", "<value>");
+
+``Updates`` may contain one of the following update operators:
+
+- ``.set()`` - replaces the value of a field with a specified one
+- ``.inc()`` - increments a field value 
+- ``.rename()`` - renames a field 
+
+See the MongoDB server manual for a :manual:`complete list of update
+operators and their usage </reference/operator/update-field/>`.
+
+Example
+~~~~~~~
+
+The store received a fresh shipment and needs to update their inventory.
+The shipment contains 20 cans of each paint color.
+
+To update the inventory, query all the colors in the ``paint_inventory``
+collection and specify the update to ``increment`` the ``qty`` field by
+``20``: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Update.java
+   :language: java
+   :dedent:
+   :start-after: begin updateManyExample
+   :end-before: end updateManyExample
+
+The method returns:
+
+.. code-block:: json   
+
+    AcknowledgedUpdateResult{ matchedCount=8, modifiedCount=8, upsertedId=null } 
+
+The following shows the documents in the ``paint_inventory`` collection:
+
+.. code-block:: json 
+
+    { "_id": {"$oid": "606dfe52d506b97c576b22bc"}, "color": "red", "qty": 25 }
+    { "_id": {"$oid": "606dfe52d506b97c576b22bd"}, "color": "purple", "qty": 28 }
+    { "_id": {"$oid": "606dfe52d506b97c576b22be"}, "color": "blue", "qty": 20 }
+    { "_id": {"$oid": "606dfe52d506b97c576b22bf"}, "color": "white", "qty": 20 }
+    { "_id": {"$oid": "606dfe52d506b97c576b22c0"}, "color": "yellow", "qty": 26 }
+    { "_id": {"$oid": "606dfe52d506b97c576b22c1"}, "color": "pink", "qty": 20 }
+    { "_id": {"$oid": "606dfe52d506b97c576b22c2"}, "color": "green", "qty": 20 }
+    { "_id": {"$oid": "606dfe52d506b97c576b22c3"}, "color": "black", "qty": 28 }
+
+If zero documents matched from the update operation, there is no change
+to the collection. You can configure an
+:doc:`upsert </fundamentals/crud/write-operations/upsert>` to attempt an
+update to a document or to insert a document if there are no matches. 
+
+.. note::
+
+    You cannot modify the ``_id`` field of a document nor change a field
+    to a value that violates a unique index constraint. See the MongoDB
+    server manual for more information on :manual:`unique indexes
+    </core/index-unique/>`.
+
+.. _replace-operation:
+
+Replace
+-------
+
+To replace a document, create a new ``Document`` with the fields and
+values that you would like to insert in your collection. This new
+document substitutes the document matched in your query filter. 
```

Comment:
Currently, it's difficult to determine what's needed to perform the replace operation from this paragraph alone. Would it make more sense to provide the information in this order instead:

1) Reiterate what the replace operation does since this is the section on the replace method.
2) Mention the required parameters (query filter and replacement document)
3) Explain how to specify/instantiate a query filter
4) Explain how to specify/instantiate a replacement document